specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/CommonDefinitions.json

```diff
@@ -0,0 +1,2876 @@
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "HybridConnectionKey resource specific properties",
+          "properties": {
+            "sendKeyName": {
+              "description": "The name of the send key.",
+              "type": "string",
+              "readOnly": true
+            },
+            "sendKeyValue": {
+              "description": "The value of the send key.",
+              "type": "string",
+              "readOnly": true
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "Identifier": {
+      "description": "A domain specific resource identifier.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Identifier resource specific properties",
+          "properties": {
+            "id": {
+              "description": "String representation of the identity.",
+              "type": "string"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "IdentifierCollection": {
+      "description": "Collection of identifiers.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Identifier"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "IpSecurityRestriction": {
+      "description": "IP security restriction on an app.",
+      "required": [
+        "ipAddress"
+      ],
+      "type": "object",
+      "properties": {
+        "ipAddress": {
+          "description": "IP address the security restriction is valid for.",
+          "type": "string"
+        },
+        "subnetMask": {
+          "description": "Subnet mask for the range of IP addresses the restriction is valid for.",
+          "type": "string"
+        }
+      }
+    },
+    "LocalizableString": {
+      "description": "Localizable string object containing the name and a localized value.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Non-localized name.",
+          "type": "string"
+        },
+        "localizedValue": {
+          "description": "Localized name.",
+          "type": "string"
+        }
+      }
+    },
+    "ManagedServiceIdentity": {
+      "description": "Managed service identity.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Type of managed service identity.",
+          "enum": [
+            "SystemAssigned",
+            "UserAssigned"
+          ],
+          "x-ms-enum": {
```

Comment:
@ruslany `type: string` removed! On purpose?

Reply:
@olydis ManagedServiceIdentity is now a flag enumeration. What is the correct swagger representation of such an enumeration?

Reply:
@naveedaz how would that enumeration be sent over the wire? I suspect there is no good Swagger representation, but assuming that you want something like `"SystemAssigned|UserAssigned"` to be sent, that's still a string so it must be modeled like that 🙂